inst/templates/SUPPORT.md

```diff
@@ -0,0 +1,29 @@
+# Getting help with {{{ Package }}}
+
+Thanks for using {{{ package }}}. Before filing an issue, please take a minute
+to look at these materials so that you can get the help you need as quickly as
+possible.
+
+The [Get help!](https://www.tidyverse.org/help/) section of the tidyverse site
+is a good place to look for general guidance, including how to make a minimal
+**repr**oducible **ex**ample (AKA a [reprex](http://reprex.tidyverse.org/)).
+
+Armed with your reprex, your next step is to figure out [where to ask](https://www.tidyverse.org/help/#where-to-ask) your question. We use GitHub 
```

Comment:
Maybe make into a bulleted list?

* If it's a question:
* If it's a bug:
* If you're not sure: